plugins/modules/mssql_script.py

```diff
@@ -52,8 +52,17 @@
       - The SQL script to be executed.
       - Script can contain multiple SQL statements. Multiple Batches can be separated by V(GO) command.
       - Each batch must return at least one result set.
-    required: true
+      - Mutually exclusive with O(script_path).
     type: str
+  script_path:
+    description:
+      - Path to a file containing the SQL script to be executed.
+      - Script can contain multiple SQL statements. Multiple Batches can be separated by V(GO) command.
+      - Each batch must return at least one result set.
+      - If the file contains a Byte Order Mark (BOM), it will be automatically detected and removed.
```

Comment:
```suggestion
      - If the file contains a Byte Order Mark (BOM), it will be automatically detected and removed.
        The return values RV(bom_removed) and RV(bom_type) will be set accordingly.
```